_posts/2020-07-28-10-Best-Practices-In-UX-UI-Design.md

```diff
@@ -0,0 +1,185 @@
+By using for example 'Employment', users might not be able to find your career page (as easily). 
+
+## 7. Breadcrumbs
+Like Hansel and Gretel, we all need to find our way back sometimes. 
+Breadcrumbs, and in our case meant specifically online, help us to do so, if done right.
+Take a look at the following example from ASOS:
+
+<img alt="ASOS uses very clear breadcrumbs" src="{{ '/img/2020-07-28-Best-Practices-In-UX-UI-Design/asos.png' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; max-width: 400px;">
+
+With breadcrumbs, users can see in the blink of an eye in which (sub)category the current page belongs. 
+The user perhaps ended up on this page by clicking a 'Recommended for you!' link and maybe he or she wants to take a look at similar items as well. 
+Being able to actually click on a (sub)category in the breadcrumbs help to do exactly that. 
+
+Even when you're just mindlessly browsing a website and you clicked on twenty different items within a category, you don't want to click the back button twenty-one times to go back to the actual results within the current category.
+Breadcrumbs make sure that your user can browse your website more easily, which creates less frustration for your user, which hopefully creates a higher conversion. 
+
+## 8. Less clicks = better
+Try to save the user clicks as much as possible. 
+Like Steve Krug describes in his [*Don't Make Me Think*](http://sensible.com/dmmt.html){:target="_blank" rel="noopener noreferrer"}, instead of only providing a tracking number for a shipment, you can add a link to the e-mail that opens the carrier's website with the tracking number already filled in.
+Another example would be to provide a Google Maps preview on the contact page instead of just providing the address.
+
+When possible, also try to mention any form errors while the user is entering the data. 
+Whether it be the phone number format or a password that is too short, anything that can be detected should immediately be communicated to the user instead of waiting until the user clicks 'Submit'. 
+
+## 9. Dropdown lists
+Dropdown lists are simple to implement and easy to use so they are present on countless websites.
+The key here is to only start using dropdown lists when we're talking about seven or more options. 
+When there are less options present, radio buttons might be a better choice for you so the user can see all the available options immediately.
+
+## 10. Include visual indicators of length and size
+Always assume that users are in a rush. 
+What users will typically do when encountering a web page that contains a lot of text, is to check how long that text actually is before starting to read it. 
+They need an idea of length or time to know if it's worth to start reading at all. 
+Medium.com does this by adding the number of minutes it takes to read their articles:
```

Comment:
Hmmm... maybe something we should also add to our Tech Blog articles! :-)

Reply:
I'm a big fan of the idea! I wonder how medium.com etc determines the reading time. I'm guessing it's just an average, but still. Something to think about! :)